api/v1beta1/keystoneapi_types.go

```diff
@@ -127,17 +127,18 @@ type KeystoneAPISpec struct {
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
 
 	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=[]
 	// NetworkAttachments is a list of NetworkAttachment resource names to expose the services to the given network
 	NetworkAttachments []string `json:"networkAttachments"`
 
 	// +kubebuilder:validation:Optional
```

Comment:
lets default this as well `// +kubebuilder:default=[]`

Reply:
Looks like the default does not work as it's treated as string?

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_placement-operator/133/pull-ci-openstack-k8s-operators-placement-operator-master-functional/1627975037304705024

Reply:
OK I think this was my mistake the proper default for empty slice is `{}`, so ` // +kubebuilder:default={}`

See: https://book.kubebuilder.io/reference/markers.html